### ISSUE TITLE
Open Collective?

### ISSUE THREAD

**Issue by pboling** [Dynamoid/Dynamoid#208]
https://opencollective.com/opensource/apply

Assuming we get donations 💰 we would be able to fund bug/feature bounties for improvements to the Dynamoid ecosystem.

CC: @andrykonchin @richardhsu

**Comment by pboling** [comment]
@bglusman Do you have experience with Open Collectives?

**Comment by bglusman** [comment]
Nope, none, but I've seen and been curious about it...  been kind of tempted to apply for [OpenPantry](https://github.com/openpantry/open-pantry) but Dynamoid could certainly be a candidate too! :-)

**Comment by pboling** [comment]
I don't know if they make exceptions, but on that apply link it is claimed that the repository needs to have at least 100 stars, for an open source project.  So I have just starred OpenPantry. :)

**Comment by andrykonchin** [comment]
Sounds good but I believe first of all we need to have a roadmap - list of missing features and known bugs, improving documentation/wiki etc. I see very little feedback from the community and it makes me sad a little.

**Comment by pboling** [comment]
Agreed.  The community is small.  But, this project is more active than many others :)

I think we can use the issue tracker to document our wishlist, bugs and todos.

We could also start using the "Projects" feature to track bigger initiatives, like perhaps the better integration with aws core libraries where this library currently duplicates functionality.

**Comment by pboling** [comment]
Open collective has been live for a few years now.  Forgot to close this.
